src/modm/board/nucleo_u575zi-q/board.hpp

```diff
@@ -0,0 +1,190 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2022, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_STM32_NUCLEO_U575ZI_Q_HPP
+#define MODM_STM32_NUCLEO_U575ZI_Q_HPP
+
+#include <modm/platform.hpp>
+#include <modm/architecture/interface/clock.hpp>
+#include <modm/debug/logger.hpp>
+
+/// @ingroup modm_board_nucleo_u575zi_q
+#define MODM_BOARD_HAS_LOGGER
+
+using namespace modm::platform;
+
+namespace Board
+{
+/// @ingroup modm_board_nucleo_u575zi_q
+/// @{
+using namespace modm::literals;
+
+/// STM32U5 running at 160 MHz generated from the
+/// internal Multispeed oscillator
```

Comment:
The comment is not correct. The system clock is generated from Hsi16, not Msi.

Reply:
Thanks for noticing